### ISSUE TITLE
Facets need indicator user has clicked them

### ISSUE THREAD

**Issue by kmonahan** [forumone/gesso#875]
The facet component is styled to look like a checkbox but is rendered as a link. When a user clicks it then, they don't see the checkbox check or uncheck until the page has reloaded. The Facets module should be creating a checkbox input, but I'm not seeing that happening on the site this is currently used on -- I suspect Facets is looking for a class name that's not in use in our templates.

One way or another, we should ensure that clicking a facet immediately checks or unchecks the checkbox.

**Comment by dcmouyard** [comment]
I believe the Facets module has a setting to output each facet as a checkbox or a link, and I’ve always used the link option, which is what our current approach in Gesso supports.

If folks are using the checkbox version, then we should probably add support for it as well.

**Comment by kmonahan** [comment]
@dcmouyard Yep, this came up on a project using the checkbox version, and interestingly enough, I've only ever used the checkbox version. So agree that supporting both would be ideal.
